test_bot/config.py

```diff
@@ -0,0 +1,31 @@
+import logging
+import os
+
+try:
+    import dotenv
+except ModuleNotFoundError:
+    print("Not loading .env")
+else:
+    dotenv.load_dotenv()
+
+
+logging.basicConfig(
+    format="%(asctime)s: [%(levelname)s] (%(threadName)s) %(name)s: %(message)s",
+    level=logging.INFO,
+)
```

Comment:
```suggestion
```
This overrides the `basicConfig` call in `__init__.py` since it runs first

Reply:
https://github.com/DisnakeDev/disnake/pull/247/commits/2afa1c270321b5f2ce4aea2aa7df869eb927ee32